tidb-binlog-architecture-evolution-and-implementation-principles.md

```diff
@@ -106,7 +106,94 @@ message Binlog {
 }
 ```
 
-For more details about binlog and the data structure definition, see [binlog.proto](https://github.com/WangXiangUSTC/tipb/blob/master/proto/binlog/binlog.proto).
+For more details about binlog and the data structure definition, see ([binlog.proto code](https://github.com/pingcap/tipb/blob/master/proto/binlog/binlog.proto)): 
```

Comment:
can change to `https://github.com/pingcap/tipb/blob/535e1abaa330653c8955cb3484261ea9f229e926/proto/binlog/binlog.proto` and remove the code

Reply:
Updated in https://github.com/pingcap/blog/pull/117/commits/aa153d4af07a42aa906df76de53f6881ae88b9c5